tests/product.spec.ts

```diff
@@ -0,0 +1,26 @@
+import {test, expect} from "@playwright/test";
+import { ApplicationPage } from "../pages/app.page";
+
+
+test('Verify user can view product details', async({page})=>{
+    const app = new ApplicationPage(page);
+    
+    const productTitle = 'Combination Pliers';
+
+    await app.home.open('/');
+
+    await expect(app.home.productsCard).not.toHaveCount(0);
+
+    const productDetails = await app.home.getProductDetails(productTitle);
+
+    await app.home.openProduct(productTitle);
+
+    await expect(page).toHaveURL(/.*\product\/.*/);
+
+    await expect((app.productDetailsPage.productName),'Product name is not visible',).toHaveText(productDetails.title);
```

Comment:
```suggestion
    await expect(app.productDetailsPage.productName,'Product name is not visible',).toHaveText(productDetails.title);
```
No need to wrap this.